### PR TITLE
Remove dupsort from unchecked table

### PR DESCRIPTION
**Pull request by SergiySW**
Currently without upgrade path (clear unchecked before or after using patch)

Part of work on #1478

### CODE REVIEW

**Comment by cryptocode** [comment]
Works great, significant speedup on my ssd test machine.

**Comment by SergiySW** [comment]
Great! Then adding upgrade path

File: rai/node/lmdb.cpp [hunk 1]
```diff
@@ -1770,18 +1770,17 @@ std::shared_ptr<rai::vote> rai::mdb_store::vote_get (rai::transaction const & tr
 std::vector<std::shared_ptr<rai::block>> rai::mdb_store::unchecked_get (rai::transaction const & transaction_a, rai::block_hash const & hash_a)
```
Comment (rkeene) [inline rai/node/lmdb.cpp]: An `unchecked_exists()` could also be helpful

Comment (SergiySW) [reply rai/node/lmdb.cpp]: unchecked_exists for key hash? Or by unchecked_key (uint256+uint256)?
If by block own hash, then we need extra lookup table

Comment (rkeene) [reply rai/node/lmdb.cpp]: unchecked_key (successor/alternate and block hash)

**Comment by rkeene** [comment]
Getting some issue with this on the beta network:

```
[beta] nano-rpc 17.0> show stats -bootstrap
Blocks:
  Count     = 3116
  Unchecked = 6584909
  Total     = 6588025
[beta] nano-rpc 17.0>
```

But there are only 5 million blocks on the beta network 😄

**Comment by rkeene** [comment]
It looks like my issue was that I didn't delete the DB beforehand (and before the upgrade path existed), dupsort=1 is still set

**Pull request merged**